guides/v2.1/config-guide/log/log-magento.md

```diff
@@ -29,7 +29,19 @@ Other channels can have a different set of handlers and logic.
 
 ## Get started with logging {#config-log-start}
 
-To start working with a logger, you must get a `\Psr\Log\LoggerInterface` instance. One way to do that is illustrated in our [database logging example]({{ page.baseurl }}/config-guide/log/log-db.html).
+To start working with a logger, you must get a `\Psr\Log\LoggerInterface` instance. With this interface, you can call the following functions to write data to log files:
+
+ * [alert()](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Logger.php#L558)
```

Comment:
Please, use links to https://github.com/php-fig/log/blob/master/Psr/Log/LoggerInterface.php instead to make the documentation more future-proof. 
Thank you

Reply:
> Please, use links to https://github.com/php-fig/log/blob/master/Psr/Log/LoggerInterface.php instead to make the documentation more future-proof.

**done**

Thank you, @rogyar 👍